app/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "lldap_app"
-version = "0.6.2-alpha"
+version = "0.7.0-alpha"
```

Comment:
Let's keep the version change out of this PR (especially since this is not breaking)